app/assets/stylesheets/pages/_howitworks.scss

```diff
@@ -1,3 +1,41 @@
+
+
+
+
+.header {
+    margin-top: 5%;
+    padding-left: 0 !important;
+}
+
+.item:hover {
+    color: gray;
+}
+
+body {
+    height: 100%;
+}
+
+.container {
```

Comment:
this is adding a margin top to every container in our app, it may look good in this page but it is interfering with everything else. give a new class to the elements you want to style